src/git-duet/git-duet-rebase/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"git-duet/internal/cmd"
+	"git-duet/internal/cmdrunner"
+)
+
+func main() {
+	err := cmdrunner.Execute(
+		cmd.New("rebase"))
```

Comment:
Because otherwise if you don't have your global git author/committer configured (which is normal when using git-duet), you get a big error saying you don't.  This is just to avoid that error when using the rebase command (same thing with cherry-pick).